geomesa-accumulo/geomesa-accumulo-dist/pom.xml

```diff
@@ -155,7 +167,7 @@
                         <phase>package</phase>
                         <configuration>
                             <attach>true</attach>
-                            <finalName>geomesa-accumulo-${project.version}</finalName>
+                            <finalName>geomesa-accumulo_${scala.binary.version}-${project.version}</finalName>
```

Comment:
could we use the placeholder for the imported deps? e.g. line 107 of this file, etc?